models/topics.model.js

```diff
@@ -23,4 +23,18 @@ exports.selectEndpoints = () => {
     .then((res) => {
         return JSON.parse(res)
     })
+}
+
+exports.selectCommentsByArticleId = (article_id) => {
+    return db.query(`SELECT comment_id, comments.votes, comments.created_at, comments.author, comments.body, article_id
+                     FROM comments 
+                     JOIN articles USING (article_id)
+                     WHERE article_id = $1
+                     ORDER BY comments.created_at DESC;`, [article_id])
+    .then(({rows}) => {
+        if (rows.length === 0) {
+            return Promise.reject({status: 404, msg: "there are no comments for this article"})
+        }
```

Comment:
If there are no comments for the article it is not an error and we would want to send the empty array back to the client.